packages/ui/src/components/Button/extendedButton.tsx

```diff
@@ -0,0 +1,50 @@
+import { Button as DefaultButton } from 'antd'
+import classNames from 'classnames'
+import React from 'react'
+
+import { MoreHorizontal } from '@open-condo/icons'
+
+import { ButtonProps } from './button'
+
+
+const BUTTON_CLASS_PREFIX = 'condo-btn'
+
+export type ExtendedButtonProps = Omit<ButtonProps, 'icon' | 'href'>
+
+const VerticalDividerSvg: React.FC = () => {
+    return (
+        <svg className={`${BUTTON_CLASS_PREFIX}-vertical-divider`} xmlns='http://www.w3.org/2000/svg' width='1' height='48' viewBox='0 0 1 48' fill='none'>
+            <rect width='1' height='48' fill='currentColor' />
```

Comment:
svg?) Why can't we just spawn a div with height: 100%?)
(just curious)

Reply:
That was easier for me)

Reply:
Hard-coded height can be a problem sometimes)
Otherwise, svg is fine

Reply:
remade with `div`